frontend/i18next.scanner/en/translation.json

```diff
@@ -59,12 +59,12 @@
   ", create the first content by clicking on the button 'Create'": ", create the first content by clicking on the button 'Create'",
   "Calendar": "Calendar",
   "Access your personal calendar": "Access your personal calendar",
-  "Spaces and notifications": "Spaces and notifications",
   "Space": "Space",
   "Spaces": "Spaces",
   "Role": "Role",
   "Notification": "Notification",
   "You are not a member of any space yet": "You are not a member of any space yet",
+  "This user is not a member of any space yet": "This member is not a member of any space yet",
```

Comment:
Why the key and the English translation are different?

Reply:
odd indeed, we are not supposed to manually edit the english translation file since they must be the translation key